pkg/printers/internalversion/printers.go

```diff
@@ -2808,6 +2819,62 @@ func printResourceQuotaList(list *api.ResourceQuotaList, options printers.Genera
 	return rows, nil
 }
 
+func printLimitRanges(limitRange *api.LimitRange, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: limitRange},
+	}
+	age := translateTimestampSince(limitRange.CreationTimestamp)
+
+	row.Cells = append(row.Cells, limitRange.Name, age)
+	if options.Wide {
+		maxItemsToShow := 2
+		showEllipsis := len(limitRange.Spec.Limits) > maxItemsToShow
+
+		for i, item := range limitRange.Spec.Limits {
+			if i >= maxItemsToShow {
+				break
+			}
+
+			formatResourceList := func(resources api.ResourceList) string {
+				parts := make([]string, 0)
+				var keys []string
+				for r := range resources {
+					keys = append(keys, string(r))
+				}
+				sort.Strings(keys)
+
+				for _, k := range keys {
+					q := resources[api.ResourceName(k)]
+					parts = append(parts, fmt.Sprintf("%s=%s", k, q.String()))
+				}
+				return strings.Join(parts, ", ")
+			}
+
+			typeStr := string(item.Type)
+			minStr := formatResourceList(item.Min)
+			maxStr := formatResourceList(item.Max)
+			row.Cells = append(row.Cells, typeStr, minStr, maxStr)
+		}
+
+		if showEllipsis {
+			row.Cells = append(row.Cells, "...")
```

Comment:
every cell needs to correspond to a column definition, this "..." would be spurious and isn't correct... if we elide content, the eliding needs to be included in one of the elided cell values